internal/version.go

```diff
@@ -0,0 +1,4 @@
+package internal
+
+// Version of executables.
+var Version = "0.1"
```

Comment:
I like that approach. Then we don't have to worry about manually editing the file every time.